example/ios/Podfile.lock

```diff
@@ -1,7 +1,7 @@
 PODS:
-  - Didomi-XCFramework (1.99.0)
+  - Didomi-XCFramework (2.0.0)
   - didomi_sdk (1.21.0):
-    - Didomi-XCFramework (= 1.99.0)
+    - Didomi-XCFramework (= 2.0.0)
```

Comment:
last iOS version is 2.0.1, you may need to run again the update script to get the very last iOS SDK version.

Reply:
This should be done automatically when running the script, as the change is not related to the APIs it should not be an issue.